src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsExchange.java

```diff
@@ -25,28 +25,27 @@
 
 package com.sun.net.httpserver;
 
-import java.io.*;
-import java.nio.*;
-import java.nio.channels.*;
-import java.net.*;
-import javax.net.ssl.*;
-import java.util.*;
+import javax.net.ssl.SSLSession;
 
 /**
- * This class encapsulates a HTTPS request received and a
- * response to be generated in one exchange and defines
- * the extensions to HttpExchange that are specific to the HTTPS protocol.
+ * This class encapsulates a HTTPS request received and a response to be
+ * generated in one exchange and defines the extensions to {@link HttpExchange}
+ * that are specific to the HTTPS protocol.
+ *
  * @since 1.6
  */
 
 public abstract class HttpsExchange extends HttpExchange {
 
-    protected HttpsExchange () {
-    }
+    /**
+     * Constructor for subclasses to call.
+     */
+    protected HttpsExchange() {}
```

Comment:
Ah. I guess this trivial change will require a CSR...

Reply:
I've created a CSR to track these changes as suggested. You can view it here: https://bugs.openjdk.java.net/browse/JDK-8254968